addon/components/bs-modal.js

```diff
@@ -186,7 +186,16 @@ export default class Modal extends Component {
    * To avoid getting out of sync it should be set only by the setter
    * of `state` property.
    */
-  @tracked isModalShown = this.state !== 'closed';
+  @tracked isInDom = this.state !== 'closed';
+
+  /**
+   * Controls if the modal is visible.
+   *
+   * It must be kept in sync with the non-tracked property `state`.
+   * To avoid getting out of sync it should be set only the setter
+   * of `state` property.
+   */
+  @tracked isShowModal = this.state === 'open' || this.state === 'opening';
```

Comment:
Can't this be a simple (auto-tracked) getter instead? The only time we set this seems to be in `set state`, but there we also set it to the same or condition, so basically it seems like it is a completely derived state, or am I missing something?